examples/typescript/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "esnext",
+        "jsx": "react",
+        "lib": [
+            "es2016",
+            "dom"
+        ],
+        "types": [
+            "node",
+            "jest"
```

Comment:
Are tests run using Jest?

Reply:
Should put on strict mode

Reply:
I agree, will put this in strict mode. 

My question is about types for global `describe` and `it` functions. is After using Jest for tests? If yes, we need types for those globals.

I was thinking instead of polluting the global namespace by adding jest to all files, we add another tsconfig.json specific to test files. we can ignore test files in the main tsconfig by a glob pattern and do the opposite in the test tsconfig. It will require the test runner to use the test tsconfig. Would that be possible?

Reply:
Agh i forgot the trick for this. It's like one line and i forgot.